docs/readthedocs/source/doc/LLM/QuickStart/install_windows_gpu.md

```diff
@@ -0,0 +1,124 @@
+
+### Install bigdl-llm
+
+* Run the commands below in Anaconda prompt. Please note that transformer version should match the model you want to use. For example, here we use transformers 4.37.0 to run the demo. 
+
+  ```bash
+  conda activate llm
+
+  pip install --pre --upgrade bigdl-llm[xpu] -f https://developer.intel.com/ipex-whl-stable-xpu
+  pip install transformers==4.37.0 
+  ```
+
+
+* Now we can test whether all the components have been installed correctly. If we can import all the packages correctly following the python file below, then the installation is correct. 
+  ```python
+  import torch
+  import time
+  import argparse
+  import numpy as np
+    
+  from bigdl.llm.transformers import AutoModel,AutoModelForCausalLM
+  from transformers import AutoTokenizer, GenerationConfig
+  ```
+
+### A quick example
+Then we use phi-1.5 as an example to show how to run the model with bigdl-llm on windows. 
+  ```python
+  import torch
+  import numpy as np
+  from bigdl.llm.transformers import AutoModelForCausalLM
+  from transformers import AutoTokenizer, GenerationConfig
+
+  PHI1_5_PROMPT_FORMAT = " Question:{prompt}\n\n Answer:"
+  generation_config = GenerationConfig(use_cache = True)
+
+  if __name__ == '__main__':
+      model_path = "microsoft/phi-1_5"
+      prompt = "What is AI?"
+      n_predict = 32
+      # Load model in 4 bit,
+      # which convert the relevant layers in the model into INT4 format
+      # When running LLMs on Intel iGPUs for Windows users, we recommend setting `cpu_embedding=True` in the from_pretrained function.
+      # This will allow the memory-intensive embedding layer to utilize the CPU instead of iGPU.
+      model = AutoModelForCausalLM.from_pretrained(model_path,
+                                                  load_in_4bit=True,
+                                                  # cpu_embedding=True,
+                                                  trust_remote_code=True)
+
+      model = model.to('xpu')
+
+      # Load tokenizer
+      tokenizer = AutoTokenizer.from_pretrained(model_path,
+                                                trust_remote_code=True)
+      
+      # Generate predicted tokens
+      with torch.inference_mode():
+          prompt = PHI1_5_PROMPT_FORMAT.format(prompt=prompt)
+          input_ids = tokenizer.encode(prompt, return_tensors="pt").to('xpu')
+          output = model.generate(input_ids, do_sample=False, max_new_tokens=n_predict, generation_config = generation_config)
+          torch.xpu.synchronize()
+          output = output.cpu()
+          output_str = tokenizer.decode(output[0], skip_special_tokens=True)
+          print('-'*20, 'Prompt', '-'*20)
+          print(prompt)
+          print('-'*20, 'Output', '-'*20)
+          print(output_str)
+   ```
+   Here is the sample output on the laptop equipped with 11th Gen Intel(R) Core(TM) i7-1185G7 and Intel(R) Iris(R) Xe Graphics after running the example program above. 

```

Comment:
How does the user run this example?

Reply:
We provide the contents of `demo.py` and users could run it as `python demo.py`.